tools/components/admin/requests/forms/types.ts

```diff
@@ -0,0 +1,21 @@
+// Address Details Object
+import { PaymentType, Province } from '@lib/graphql/types';
+
+// Address
+export type AddressDetailsData = {
+  isAddressSameAsHome: boolean;
+  addressFullName: string | null;
+  addressLine1: string | null;
+  addressLine2: string | null;
+  city: string | null;
+  province?: Province | null;
+  postalCode: string | null;
+};
+
+// Payment Details Data Object
+export type PaymentDetailsData = {
+  paymentMethod: PaymentType | undefined;
+  donationAmount: number | null;
+  shippingAddressData: AddressDetailsData;
+  billingAddressData: AddressDetailsData;
+};
```

Comment:
As per our discussion at last sync, please flatten the `PaymentDetailsData` object and use TypeScript `Pick` to pick fields from the existing `Application` type